internal/cli/atlas/dbusers/create.go

```diff
@@ -183,7 +183,7 @@ func CreateBuilder() *cobra.Command {
   $ %[1]s dbuser create --username <username> --role clusterMonitor,backup --projectId <projectId>
 
   Create a database user with multiple scopes:
-  $ %[1]s dbuser create --username <username> --role clusterMonitor --scope clusterName:CLUSTER,DataLakeName:DATA_LAKE --projectId <projectId>`,
+  $ %[1]s dbuser create --username <username> --role clusterMonitor --scope <REPLICA-SET ID>,<storeName> --projectId <projectId>`,
```

Comment:
[q] is store name meant to be a datalake?

Reply:
For context this flag would be `--scope <clusterName>[:CLUSTER]` or `--scope <datalakeName:DATALAKE>`
where the scope is always a `name` of a thing and the type of a thing where the type is optional for clusters